packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -303,6 +326,10 @@ class InputDecoration {
       description.add('suffixText: $suffixText');
     if (suffixStyle != null)
       description.add('suffixStyle: $suffixStyle');
+    if (counterText != null)
+      description.add('counterText: $counterText');
+    if (counterStyle != null)
+      description.add('counterStyle: $counterStyle');
     return 'InputDecoration(${description.join(', ')})';
   }
```

Comment:
at some point soon we're really going to have to turn this into a Diagnosticable